source/_posts/2015-03-23-artsy-technology-stack-2015.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Artsy's Technology Stack, 2015
+date: 2015-03-23
+comments: true
+categories: [Technology]
+author: Daniel Doubrovkine
+github-url: https://www.github.com/dblock
+twitter-url: http://twitter.com/dblockdotorg
+blog-url: http://code.dblock.org
+---
+Artsy has now grown past 100 team members and our Engineering organization is now 20 strong. For a brief overview of what the company has accomplished in the last two years, check out our [2013](http://2013.artsy.net) and [2014](http://2014.artsy.net) reviews.
+
+This is a good opportunity to describe our updated technology stack. Last time [we did this](/blog/2012/10/10/artsy-technology-stack) was when Artsy launched publicly in 2012.
+
+<center><img src='/images/2015-03-23-artsy-technology-stack-2015/stats.png'></center>
+
+Three years ago Artsy was a classic Ruby-on-Rails monolith with a handful of adjacent processes and tools. We've since broken it up into many independent services, and continue to heavily be a Ruby and JavaScript shop, using Rails where appropriate, with native code on mobile devices and some JVM-based experiments in micro-services.
+
+<center><img src='/images/2015-03-23-artsy-technology-stack-2015/languages.png'></center>
+
+<!-- more -->
+
+What you see today when you go to [www.artsy.net](https://www.artsy.net) is a website built with [Ezel.js](http://ezeljs.com), which is a boilerplate for Backbone projects running on Node.js and using Express and Browserify. The [CoffeeScript code](https://github.com/artsy/force-public) is open-source. The mobile version of www.artsy.net is [m.artsy.net](https://m.artsy.net) and is built on the same technology. Both run on Heroku and use [Redis](http://redis.io) for caching.
```

Comment:
ps Since React is so hot right now, any reason to mention its use in a few internal apps? (Writer and Helix?) cc reactfans @craigspaeth @sweir27